src/families/FeeField/BitcoinKind.js

```diff
@@ -48,6 +48,7 @@ const FeesField = ({ transaction, account, onChange, status }: Props) => {
 
   const bridge = getAccountBridge(account)
   const { feePerByte, networkInfo } = transaction
+  let input: ?HTMLInputElement
```

Comment:
You should prefer using a ref.

useRef and pass that ref to ref prop.
Check react doc

Reply:
Gotcha